CollAction/Views/Shared/_Layout.cshtml

```diff
@@ -1,11 +1,20 @@
-﻿<!DOCTYPE html>
+﻿@using Microsoft.AspNetCore.Http.Features
+
+@{
+    var consentFeature = Context.Features.Get<ITrackingConsentFeature>();
+    var canTrack = consentFeature?.CanTrack ?? false;
```

Comment:
I don't think you need the null coalesce here, if the consent feature isn't set, it should be an error right?